doc/main.html

```diff
@@ -789,6 +789,39 @@ <h4>Lists of key-value pairs</h4>
 
       </section>
 
+      <section id="sec-list-element">
+        <h4>Lists</h4>
+
+        <p>A list can be inserted using either an <code>ol</code> or <code>ul</code> element. The <code>ul</code> (unordered list) element should be used when the order of listed items is not meaniful or important. The <code>ol</code> (ordered list) element should be only used when the order of listed items is meaningful and/or important to the list and/or document.</p>
+
+        <p>An <code>ol</code> or <code>ul</code> element must contain:</p>
+        <ul>
+          <li>one or more <code>li</code> element</li> 
```

Comment:
```suggestion
          <li>two or more <code>li</code> element</li> 
```

We should discourage single items, just like we prohibit single sections.

If you agree, I can modify the validator.

Can you add unit tests?

Reply:
Agreed. I'll update the wording as well.

Reply:
Thinking more about, I do not think we should prohibit single items since a list might be used to represent a tree where a node might have one leaf.